lib/analytics.js

```diff
@@ -791,12 +821,43 @@ Analytics.prototype._invoke = function(method, facade) {
                 result = new Facade(result);
               }
 
-              metrics.increment('analytics_js.integration.invoke', {
-                method: method,
-                integration_name: integration.name
-              });
+              // apply destination middlewares
+              // Apply any integration middlewares that exist, then invoke the integration with the result.
+              if (self._destinationMiddlewares[integration.name]) {
+                self._destinationMiddlewares[integration.name].applyMiddlewares(
+                  facadeCopy,
+                  integration.name,
+                  function(result) {
```

Comment:
is there a way, using JSDocs, document the format we expect `result` to respect?

Reply:
I think you would need to do one line per property like this https://jsdoc.app/tags-param.html#parameters-with-properties